tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1385,6 +1424,23 @@ Status Converter::BuildCudaEngine(
   TrtUniquePtrType<nvinfer1::IBuilderConfig> builder_config(
       trt_builder_->createBuilderConfig());
   builder_config->setMaxWorkspaceSize(max_workspace_size_bytes);
+
+#if IS_TRT_VERSION_GE(7, 1, 3, 0)
+   // Only activate TRT deterministic mode if there is a requested tactic ID
+   static bool trt_debug_deterministic = [] {
+     int64 trt_tactic_id = -1;
```

Comment:
Let's rewrite this line to the following, note we remove the initialization as it is not necessary:
int64 algorithm_id;